modules/031-linstor/monitoring/prometheus-rules/linstor-affinity-controller.yaml

```diff
@@ -0,0 +1,36 @@
+- name: kubernetes.linstor.affinity-controller_state
+  rules:
+    - alert: D8LinstorAffinityControllerPodIsNotReady
+      expr: min by (pod) (kube_pod_status_ready{condition="true", namespace="d8-linstor", pod=~"linstor-affinity-controller-.*"}) != 1
+      labels:
+        severity_level: "6"
+        tier: cluster
+      for: 30m
```

Comment:
```
for: 30m
```
What is a rationale behind such a long waiting period for a not-ready pod?

Reply:
This copy/paste came from extended-monitoring module, updated it to 30m/2m respectively

Reply:
I see 10m and 2m. For me looks much better than 30m!